### ISSUE TITLE
Quick Replies

### ISSUE THREAD

**Issue by zanstro** [Schmavery/facebook-chat-api#822]
Is there any way to interact with quick replies? I want to be able to read a quick reply and then imitate clicking on it. Is this currently possible? Thank you in advance.

**Comment by returnmk** [comment]
> Is there any way to interact with quick replies? I want to be able to read a quick reply and then imitate clicking on it. Is this currently possible? Thank you in advance.

I don't think so... I am looking to do that too but can't find any workaround...

**Comment by marcelo-bianchi** [comment]
write your quick reply command with empty empty content on facebook

**Comment by ryancalo** [comment]
> write your quick reply command with empty content on facebook side
> then with the bot read the content and just reply what you want to

what do you mean by this
